index.js

```diff
@@ -36,7 +36,7 @@ const adjustRenderer = directory => {
     let path = request.url.substr(isWindows ? 8 : 7)
```

Comment:
I don't fully understand why this is being done. I suspect it has something to do with the fact that on windows a `file://` protocol has an extra slash `/`, but I couldn't find any docs about this in a cursory search.